contextvh/src/main/java/contextvh/ContextEntity.java

```diff
@@ -30,6 +36,7 @@
 	 */
 	public ContextEntity(final EntityListener env, final String intendedStakeholder, final Integer slotID) {
 		super(env, intendedStakeholder, slotID);
+		customActions.addAction(perceptsFilter);
```

Comment:
@spiderbiggen on second thought, this is possible since `ActionContainer` extends Hashmap.
See 57e1e15